javascript/builtins/RegExp.json

```diff
@@ -1810,6 +1810,61 @@
             }
           }
         },
+        "@@matchAll": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp/@@matchAll",
+            "spec_url": "https://tc39.github.io/proposal-string-matchall/#sec-regexp-prototype-matchall",
+            "support": {
+              "chrome": {
+                "version_added": "73"
+              },
+              "chrome_android": {
+                "version_added": "73"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "60"
+              },
+              "opera_android": {
+                "version_added": true
```

Comment:
Is there a reason this one is `true` instead of 60? The Chrome status page says 60, at least

Reply:
This would seem to apply to the others Opera for Android entries in this PR too

Reply:
Yeah, Chromestatus is wrong about Opera for Android. Opera Android 49 is the latest known version and so they are far away from having 60. See https://github.com/mdn/browser-compat-data/pull/1712#issuecomment-416886047 (opera android data is wrong across our data thanks to chromestatus and other sources. I don't know how to deal with this yet).

Reply:
Ah, got it. Thanks for the update!